lib/galaxy/managers/citations.py

```diff
@@ -47,7 +47,7 @@ def _raw_get_bibtex(self, doi):
         doi_url = "https://doi.org/" + doi
         headers = {'Accept': 'text/bibliography; style=bibtex, application/x-bibtex'}
         req = requests.get(doi_url, headers=headers)
-        return req.text
+        return req.content.decode("utf-8")
```

Comment:
```suggestion
        req.encoding = req.apparent_encoding
        return req.text
```
might be a better approach here (https://2.python-requests.org/en/master/api/#requests.Response.apparent_encoding).

Reply:
Yeah, just found the same [here](https://stackoverflow.com/a/52615216) . ```apparent_encoding``` looks better :thumbsup: